lib/queue.js

```diff
@@ -222,7 +222,7 @@ class Queue extends Emitter {
       this._isClosed = true;
 
       const clients = [];
-      if (this.settings.quitCommandClient) {
+      if (this.settings.quitCommandClient && this.client) {
```

Comment:
worth unit testing this?

Reply:
Good call! Found another bug :) 3183887d50d6c37533223a6eeecb17cac003e859